### DISCUSSION TITLE
PCLMainPage

### DISCUSSION THREAD

**Discussion by awa-qwq-TAT** [Meloong-Git/PCL#5228] [自定义主页]
名字：PCLMainPage
链接：https://raw.gitcode.com/awa-qwq-TAT/PCLMainPage/raw/newest/Custom.xaml

**Comment by allMagicNB** [comment]
@LTCatt 有没兴趣对主页命名做点限制？（

**Comment by MoYuan-CN** [comment]
## 可用性检查报告

提供主页 URL:
https://raw.gitcode.com/awa-qwq-TAT/PCLMainPage/raw/newest/Custom.xaml
检查主页是否可用
✅ 正常
检查开源版本 PCL 是否可用
✅ 正常
检查是否设置版本号(.ini)
✅ 正常
检查是否设置 UA 过滤
❌ 未设置
检查是否设置 Referer 过滤
❌ 未设置

###### Powered by [Light-Beacon/PCLHomepageChecker](https://github.com/Light-Beacon/PCLHomepageChecker)

**Comment by MoYuan-CN** [comment]
## 综合反馈
> 参见 https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-9641112

您的主页过于简单，根据现有情况，龙猫的意见是
如果已经有一个相同类型的主页，除非新提交的质量明显比另一个高，不然都不考虑……

建议您考虑往指定一个方向专门发展

<details>
<summary>仅针对内容的建议</summary>

![image](https://github.com/user-attachments/assets/c275680d-67e7-46a7-b190-a94c2465e27f)
实际上没有对应版本时无法启动，实用性较低

![image](https://github.com/user-attachments/assets/58fd60bc-e296-4ab3-8cf6-062fa4d65d0b)
等宽一下？

![image](https://github.com/user-attachments/assets/0cc8d116-e315-4af0-b818-653a77d8fd0a)
指向的链接报告使用了不受支持的协议（ERR_SSL_VERSION_OR_CIPHER_MISMATCH）
额外 Tip：观察到链接指向了 `.rth1.xyz`，RetieBox 已于 <不知道什么时候> 将二级域名切换为了 `rth5.com`

</div>

**Comment by Light-Beacon** [comment]
> 不包含服务器、第三方网站的广告性质链接，间接广告也不行

<details>
<summary>补：截图</summary>
<img width="1242" alt="image" src="https://github.com/user-attachments/assets/fd705314-abde-471e-bf3a-f10bc43ea773">
</details>

**Reply by Joker2184** [reply]
6

**Reply by awa-qwq-TAT** [reply]
所以要删掉服务器模块，不能让玩家自己上传服务器？

**Reply by tangge233** [reply]
> 所以要删掉服务器模块，不能让玩家自己上传服务器？

如果主页是为你的服务器设计的，应该直接使用 `联网更新` 而不是尝试加入预设……

**Reply by awa-qwq-TAT** [reply]
> > 所以要删掉服务器模块，不能让玩家自己上传服务器？
> 
> 如果主页是为你的服务器设计的，应该直接使用 而不是尝试加入预设……`联网更新`

那如果是让玩家自己上传的服务器呢（

**Reply by Joker2184** [reply]
> > > 所以要删掉服务器模块，不能让玩家自己上传服务器？
> > 
> > 
> > 如果主页是为你的服务器设计的，应该直接使用 而不是尝试加入预设……`联网更新`
> 
> 那如果是让玩家自己上传呢（

也属于

因为这个在预设 有这是 PCL推荐的服务器  龙猫认为好的服务器 的嫌疑

所以不能含有任何服务器内容

然后

> 我目前的标准是，如果已经有一个相同类型的主页，除非新提交的质量明显比另一个高，不然都不考虑……
所以最好别和已有类型的撞车，抱歉……

此主页已有相同类型主页 所以应该不会考虑

**Reply by Joker2184** [reply]
预设 简单主页

**Reply by awa-qwq-TAT** [reply]
> 预设 简单主页

里面没有啊（悲）

**Reply by Joker2184** [reply]
![QQ截图20241214193924](https://github.com/user-attachments/assets/1672091b-05f1-4b4f-bd64-28f7ef4957ad)
这个...？
要不先更新下PCL..?

**Reply by awa-qwq-TAT** [reply]
![屏幕截图 2024-12-14 194511](https://github.com/user-attachments/assets/56ebadaf-631e-422e-9c6b-dbc8bfc133bf)
![屏幕截图 2024-12-14 194519](https://github.com/user-attachments/assets/7d518172-964e-40a8-a835-0e647c475a6d)
嗯，没错啊

**Reply by Joker2184** [reply]
功能差不多啊

大概不会考虑

**Reply by Joker2184** [reply]
> > > > > 所以要删掉服务器模块，不能让玩家自己上传服务器？
> > > > 
> > > > 
> > > > 如果主页是为你的服务器设计的，应该直接使用 而不是尝试加入预设……`联网更新`
> > > 
> > > 
> > > 那如果是让玩家自己上传呢（
> > 
> > 
> > 也属于
> > 因为这个在预设 有这是 PCL 推荐的服务器 龙猫认为好的服务器 的嫌疑
> > 所以不能含有任何服务器内容
> > 然后
> > > 我目前的标准是，如果已经有一个相同类型的主页，除非新提交的质量明显比另一个高，不然都不考虑…… 所以最好别和已有类型的撞车，抱歉......
> > 
> > 
> > 此主页已有相同类型主页 所以应该不会考虑
> 
> 我是说我可以考虑向服务器方向发展，小工具什么的算是顺带的 ~~ 所以我还是没找到你说的服务器在哪里~~

~~我刚表述可能有点问题 给你带来了误解~~ 指的是如果除开 服务器这个栏目 定位过于相似也不会考虑 :D

**Reply by awa-qwq-TAT** [reply]
> > > > > > 所以要删掉服务器模块，不能让玩家自己上传服务器？
> > > > > 
> > > > > 
> > > > > 如果主页是为你的服务器设计的，应该直接使用 而不是尝试加入预设……`联网更新`
> > > > 
> > > > 
> > > > 那如果是让玩家自己上传呢（
> > > 
> > > 
> > > 也属于
> > > 因为这个在预设 有这是 PCL 推荐的服务器 龙猫认为好的服务器 的嫌疑
> > > 所以不能含有任何服务器内容
> > > 然后
> > > > 我目前的标准是，如果已经有一个相同类型的主页，除非新提交的质量明显比另一个高，不然都不考虑……所以最好别和已有类型的撞车，抱歉......
> > > 
> > > 
> > > 此主页已有相同类型主页 所以应该不会考虑
> > 
> > 
> > 我是说我可以考虑向服务器方向发展，小工具什么的算是顺带的 ~~ 所以我还是没找到你说的服务器在哪里~~
> 
> ~我刚表述可能有点问题 给你带来了误解~ 指的是如果除开 服务器这个栏目 定位过于相似也不会考虑 :D

o~ thank you~

**Comment by LTCatt** [comment]
https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-9641112

**Reply by CreeperIsASpy** [reply]
那主页命名方面也考虑限制一下？
cc @LTCatt
